conda_smithy/utils.py

```diff
@@ -12,6 +12,15 @@
 import ruamel.yaml
 
 
+def get_feedstock_name_from_meta(meta):
+    """Resolve the feedtstock name from the parsed meta.yaml."""
+    if "parent_recipe" in meta.meta["extra"]:
+        return meta.meta["extra"]["parent_recipe"]["name"]
+    elif "feedstock_name" in meta.meta["extra"]:
```

Comment:
Shouldn't this come before? `parent_recipe` is defined for all outputs except for the top level package.